jsonrpc/types.go

```diff
@@ -102,6 +102,7 @@ type block struct {
 	Hash            types.Hash          `json:"hash"`
 	Transactions    []transactionOrHash `json:"transactions"`
 	Uncles          []types.Hash        `json:"uncles"`
+	BaseFee         argUint64           `json:"baseFee,omitempty"`
```

Comment:
Can we test this with a use case in the `testsuite`?

Reply:
I mean, test that it is not marshaled if zero.

Reply:
Done